distribution/src/main/resources/docker-compose/docker-compose.yml

```diff
@@ -1,9 +1,12 @@
-version: "2"
+version: "3.8"
```

Comment:
I am not sure if that is compatible with GHA env setup - but assuming that this docker-compose is used in pipeline and tests are green.

Reply:
This d-c is not used in pipeline - it's used only for local run of the app